wizard/WizardModeSelection.qml

```diff
@@ -0,0 +1,134 @@
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 0 * scaleRatio
+
+            WizardHeader {
+                title: qsTr("Mode selection.") + translationManager.emptyString
+                subtitle: qsTr("Please select the statement that best matches you.") + translationManager.emptyString
+            }
+
+            WizardMenuItem {
+                Layout.topMargin: 20 * scaleRatio
+                headerText: qsTr("Simple mode") + translationManager.emptyString
+                bodyText: qsTr("Easy access to sending, receiving and basic functionality.") + translationManager.emptyString
+                imageIcon: "../images/remote-node.png"
+
+                onMenuClicked: {
+                    appWindow.changeWalletMode(0);
+                    wizardController.wizardState = 'wizardModeRemoteNodeWarning';
+                }
+            }
+
+            Rectangle {
+                Layout.preferredHeight: 1
+                Layout.topMargin: 5 * scaleRatio
+                Layout.bottomMargin: 10 * scaleRatio
+                Layout.fillWidth: true
+                color: MoneroComponents.Style.dividerColor
+                opacity: MoneroComponents.Style.dividerOpacity
+            }
+
+            WizardMenuItem {
+                headerText: qsTr("Simple mode") + " (bootstrap)" + translationManager.emptyString
+                bodyText: qsTr("Easy access to sending, receiving and basic functionality. The blockchain is downloaded to your computer.") + translationManager.emptyString
+                imageIcon: "../images/local-node.png"
+
+                onMenuClicked: {
+                    appWindow.changeWalletMode(1);
+                    wizardController.wizardState = 'wizardModeBootstrap';
+                }
+            }
+
+            Rectangle {
+                Layout.preferredHeight: 1
+                Layout.topMargin: 5 * scaleRatio
+                Layout.bottomMargin: 10 * scaleRatio
+                Layout.fillWidth: true
+                color: MoneroComponents.Style.dividerColor
+                opacity: MoneroComponents.Style.dividerOpacity
+            }
+
+            WizardMenuItem {
+                headerText: qsTr("Advanced mode") + translationManager.emptyString
+                bodyText: qsTr("Includes extra features like mining and message verification. The blockchain is downloaded to your computer.") + translationManager.emptyString
+                imageIcon: "../images/local-node-full.png"
+
+                onMenuClicked: {
+                    appWindow.changeWalletMode(2);
+                    wizardController.wizardState = 'wizardHome';
+                }
+            }
+
+            WizardNav {
+                Layout.topMargin: 5 * scaleRatio
+                btnPrevText: qsTr("Change language") + translationManager.emptyString
+                btnNext.visible: false
+                progressSteps: 0
+
+                onPrevClicked: {
```

Comment:
Suggestion: This Mode Selection screen could use a "previous" button, along with the change language button.

Reply:
Where would previous bring you?

Reply:
I guess back to the main menu?

Reply:
It would add some complexity to determine if an user already has already chosen a mode, in which case you can show the previous button but not on the first run. I would prefer to leave it like it is.

Reply:
One more vote for a *Back to menu* button on the *Mode selection* screen: If you arrive here from the menu it's impossible to back out without making a choice, with the added difficulty that right now this screen does not indicate which mode is the currently active one.